src/cheri_types.sail

```diff
@@ -94,7 +93,7 @@ function CapExCode(ex) : CapEx -> bits(5) =
     CapEx_ReturnTrap                    => 0b00110,
     CapEx_TSSUnderFlow                  => 0b00111,
     CapEx_UserDefViolation              => 0b01000,
-    CapEx_TLBNoStoreCap                 => 0b01001,
```

Comment:
At some point we will want to annotate this CHERI exception code as MIPS-only in the ISA doc.